_sources/rust-2019-more-librarians.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Rust 2019 (7): More librarians"
+---
+
+# Rust 2019 (7): More librarians
+
+Rust - the project - has seen a huge growth in the last year. Rust - the community - as well. This leads to two effects: while in 2017, it was easy to keep track of everything the project does, now, following even single actions can be very time-consuming. This lets the project drift apart a little. It also lets the community drift apart a little, with some tensions between groups becoming more apparant.
+
+Lots of the project success is based on trust: trust between the working groups, trust of the community. This is easier to maintain in smaller circles then in larger ones. Trust is hard to gain but easy to lose. But we also face the problem that we communicate over a problematic medium.
+
+withoutboats has [pointed out nicely](TODO: Link) that consuming community input has become a huge problem. I like to focus on another issue: it has also become a problem for project members to consume and track their own projects activity. Similarly, it has become a much bigger problem for the interested community member to follow the project, even when they do Rust on the job!
+
+This builds tension: the community feels worse informed than before and project members cannot cope with that, as they are also less and less well informed. This tension can be seen in frequent accusations of certain teams just "doing what they want" or that the project listens less and less to the community. While I frequently catch myself calling people out for baseless accusations of that kind, I also maintain that we have to appreciate that _such feelings are valid_ and should not be dealt with by shutting them down.
+
+I have a couple of potential fixes to this problem:
+
+* Better, more central documentation of team action and decisions: Currently, every team maintains their own tech stack. There's no necessity to file conclusions or approaches in a centralised fashion. The burden of tracking down info is on the searching person.
+* Clearer, better consumable history of action: While [This week in Rust](Link: TODO) is a good resource, it still makes it hard to disambiguate what information is relevant to whom.
+* Clearer team policies: how can you join a team, how can you participate in their meetings as bystander and what can they support users with?
```

Comment:
wasm workgroup has public weekly meetings where everyone can join. its pretty good.

Reply:
Most team meetings are public, but the policies are not unified and it's hard to figure out easily. I'll make that point clearer. Thanks :).